main.go

```diff
@@ -337,6 +337,9 @@ func dump() error {
 			saver := &JSONFilesHistorySaver{Dirpath: config.OutDirPath}
 			saver.SaveAccount(*me)
 			log.Info("User Account Info Saved")
+
+			tgSaveUserProfilePhotos(tg, *me, saver.profilePicFPath())
```

Comment:
There should be a config option for profile pics, so the whole dump could still be text-only. Something like `"dump_account_profilepics": true/false` and disabled by default.

That way it can also work independently of `config.DoAccountDump` (i.e. outside that `if`).